framework/auth/views.py

```diff
@@ -1035,11 +1035,14 @@ def external_login_email_get():
 
     form = ResendConfirmationForm(request.form)
     session = get_session()
-    if not session.is_external_first_login:
+    if session.get('auth_user_external_first_login', False) is not True:
```

Comment:
Fix 1:

`is_external_first_login()` is a deprecated method in the old `Session` model. In addition, need to explicitly check `True` instead of `True-ish`.